containers/kubernetes/reference-content/kubernetes-node-images.mdx

```diff
@@ -0,0 +1,52 @@
+---
+meta:
+  title: Kubernetes node images
+  description: This page provides information about Kubernetes node images for your Kubernetes cluster
+content:
+  h1: Kubernetes node images
+  paragraph: This page provides information about Kubernetes node images for your Kubernetes cluster
+tags: kubernetes load-balancer loadbalancer service annotations ip expose
+dates:
+  validation: 2024-01-18
+  posted: 2024-01-18
+categories:
+  - kubernetes
+---
+
+Kubernetes node images refer to the containerized operating system and runtime environment that run on each node within a Kubernetes cluster.
+These images encapsulate the essential components necessary for hosting and managing containers, such as the container runtime (e.g. `containerd`), the kubelet (Kubernetes agent), and other dependencies.
+
+In Kubernetes, a node is a physical or virtual machine inside a cluster, that serves for deploying and running containerized workloads.
+A Kubernetes cluster consists of a main node (control plane) and multiple worker nodes. These worker nodes are where the actual containers run.
+
+## What does Scaleways node images contain?
+
+Scaleway Kubernetes node images are customized Ubuntu Linux-based OS images, specifically built for Scaleway's Kubernetes infrastructure.
+These images include `containerd`, a high-performance container runtime, and Scaleway's Kubernetes Kapsule agent, facilitating communication between the Kubernetes control plane and the underlying infrastructure.
+
+## What are the benefits of the new node image system?
+
+The new node images offers advantages in cluster consistency, management ease, and security. 
+Our images are optimized for performance and compatibility with the Kubernetes control plane and integrate seamlessly with the Kubernetes ecosystem.
+
+The main benefits of using the new node images are:
+- Accelerated upgrade timelines
+    - Uninterrupted service continuity with hot upgrades
+    - Increased upgrade frequency for large clusters
```

Comment:
the increase upgrade frequency is our hope, not directly a benefit. It's because the upgrade is faster we HOPE users will upgrade more often :D but it remains their decision/their action in the end!

Reply:
```suggestion
```